runtime/parachains/src/configuration/migration.rs

```diff
@@ -137,32 +135,32 @@ pub mod v3 {
 		}
 	}
 
-	pub struct MigrateToV3<T>(sp_std::marker::PhantomData<T>);
-	impl<T: Config> OnRuntimeUpgrade for MigrateToV3<T> {
+	pub struct MigrateToV4<T>(sp_std::marker::PhantomData<T>);
```

Comment:
I don't see this hooked into the runtimes Migrations.

Reply:
Yes, it's not. We discussed that [here](https://github.com/paritytech/polkadot/pull/6345/files#r1050765851).

I don't know in what order and on what networks we will run the migration so I haven't added it anywhere yet.

Reply:
We should it add it to every network (polkadot, kusama, westend), because they will need to run the migration eventually. For Polkadot, for example, it the migration was added to some runtime that isn't applied yet and removed in master, because that runtime will be applied eventually.

Reply:
Done